src/com/workspace/utill/LogWriter.java

```diff
@@ -1,22 +1,22 @@
 package com.workspace.utill;
-
+import com.workspace.io.*;
 public class LogWriter {
+    private ConsoleWriter consoleWriter = null;
```

Comment:
If I am not wrong, these consoleWriter and fileWriter objects will be created only once. Can we make it static ?